### ISSUE TITLE
Change SDK separate handling for v11/v12+ GTM objects

### ISSUE THREAD

**Issue by jasonrahm** [F5Networks/f5-common-python#1497]
Currently, if users have v11 and v12 devices they need to manage, they need to write separate functions as we don't provide a single method to handle both cases. This is broken.

[Reference](https://devcentral.f5.com/questions/handling-differences-in-v11-v12-in-python-sdk-for-icontrol-rest-61020#comment68259)

**Comment by wojtek0806** [comment]
the whole interface was setup this way, the way our classes work is that they are chained to roughly reflect the REST URI, since v11 and v12 interfaces changed drastically this is reflected in the modules. I would suggest doing something that is not going to change the current class structures for those endpoints i.e. an external utility rather than SDK change

**Comment by niall-munnelly** [comment]
Thanks for this; we have a clearer understanding of what to expect using the SDK, now.

**Comment by jasonrahm** [comment]
Fair enough, @wojtek0806. I'll look at building a utility we can include.
